src/CBot/CBotInstr/CBotExprLitString.cpp

```diff
@@ -16,6 +16,7 @@
  * You should have received a copy of the GNU General Public License
  * along with this program. If not, see http://gnu.org/licenses
  */
+#include <stdexcept>
```

Comment:
I think this will cause the linter to complain.
#includes with ```<...>``` angle brackets should be placed after #includes with ```"..."``` quotes and in alphabetical order.

Reply:
Fixed by ammending the commit.